.travis.yml

```diff
@@ -6,4 +6,6 @@ matrix:
       script: ./sbt ++$TRAVIS_SCALA_VERSION clean test mimaReportBinaryIssues
 
     - scala: 2.11.8
-      script: ./sbt ++$TRAVIS_SCALA_VERSION clean test mimaReportBinaryIssues
+      script: ./sbt ++$TRAVIS_SCALA_VERSION clean coverage test coverageReport mimaReportBinaryIssues
```

Comment:
Are we doing this for 2.11 only because the results should be identical for 2.10?